newlib/libc/sys/arc/syscalls.c

```diff
@@ -35,6 +37,10 @@ _syscall2 (int,gettimeofday, struct timeval *,tv, void *,tz)
 
 _syscall1 (void,exit, int,ret)
 
+_syscall1 (_CLOCK_T_,times, struct tms *,buf)
+
+_syscall2 (int,stat, const char *,file, struct stat *,st)
```

Comment:
The same observation. You can use the gcc's check_GNU_style.sh script to see if your patch is GNU style consistent. Although the script doesn't guarantee this 100%.